app/src/main/java/com/example/tasky/MainActivity.kt

```diff
@@ -22,7 +24,16 @@ class MainActivity : ComponentActivity() {
                     modifier = Modifier.fillMaxSize(),
                     color = MaterialTheme.colorScheme.background
                 ) {
-                    LoginScreen()
+                    LoginScreen(
+                        loginState = LoginState(
```

Comment:
You have a LoginViewModel

Why not use the state from there?

Reply:
I wanted to leave the creation and the connection with the viewModel later